docs/developer-guide/etendo-classic/how-to-guides/how-to-create-client-event-handler-actions.md

```diff
@@ -0,0 +1,125 @@
+---
+tags:
+  - client event handler
+  - actions
+  - Etendo Classic
+---
+
+#  How to Create Client Event Handler Actions
+  
+##  Overview
+
+  This section discusses how to implement client-side functions (JavaScript) that are executed before or after an event is fired in a standard window of the User Interface.
```

Comment:
The overview section speak about javascripts, but only java process are develop.

Reply:
@ivanrobledofutit Don't know if you see this comment. I @ you just in case.